src/dataframe/io.jl

```diff
@@ -801,7 +801,7 @@ function readtable(io::IO,
                    separator::Char = ',',
                    quotemark::Vector{Char} = ['"'],
                    decimal::Char = '.',
-                   nastrings::Vector = ["", "NA"],
+                   nastrings::Vector = ["", "NULL", "NA"],
```

Comment:
Is this strictly necessary? Do we actually write out `NULL` now in writetable? If we do, I'd say we shouldn't and just follow the typical CSV/DBMS tradition of writing nothing for null fields. i.e `col1,col2,col3\n,,\n`.